ingestors/kafka/src/main/scala/hydra/kafka/algebras/MetadataAlgebra.scala

```diff
@@ -30,14 +29,14 @@ object MetadataAlgebra {
   final case class TopicMetadataContainer(key: TopicMetadataV2Key, value: TopicMetadataV2Value, keySchema: Option[Schema], valueSchema: Option[Schema])
 
   def make[F[_]: Sync: Concurrent: Logger](
-                        metadataTopicName: TopicName,
-                        consumerGroup: ConsumerGroup,
-                        kafkaClientAlgebra: KafkaClientAlgebra[F],
-                        schemaRegistryAlgebra: SchemaRegistry[F],
-                        consumeMetadataEnabled: Boolean
-                      ): F[MetadataAlgebra[F]] = {
+                                            metadataTopicName: Subject,
+                                            consumerGroup: ConsumerGroup,
+                                            kafkaClientAlgebra: KafkaClientAlgebra[F],
+                                            schemaRegistryAlgebra: SchemaRegistry[F],
+                                            consumeMetadataEnabled: Boolean
+                                          ): F[MetadataAlgebra[F]] = {
     val metadataStream: fs2.Stream[F, (GenericRecord, Option[GenericRecord])] = if (consumeMetadataEnabled) {
-      kafkaClientAlgebra.consumeMessages(metadataTopicName, consumerGroup, commitOffsets = false).map(record => (record._1, record._2))
+      kafkaClientAlgebra.consumeMessages(metadataTopicName.toString, consumerGroup, commitOffsets = false).map(record => (record._1, record._2))
```

Comment:
I think you will need to use `metadataTopicName.value` here because I think .toString includes some extra garbage like the class name in it

Reply:
Done.